pkgs/by-name/ne/neovim-unwrapped/package.nix

```diff
@@ -113,6 +114,11 @@ stdenv.mkDerivation (
       ./system_rplugin_manifest.patch
     ];
 
+    patchPhase = ''
+      # loosen the version requirement for wasmtime to use the nixpkgs version
+      sed -i -E 's/find_package\(Wasmtime [0-9.]+ EXACT REQUIRED\)/find_package\(Wasmtime REQUIRED\)/' ./src/nvim/CMakeLists.txt
```

Comment:
Any reason not to use `substituteInPlace ... --replace-fail ...` instead?

Reply:
`substituteInPlace` does not support regexes, IIRC.

Reply:
True!